ConnectionManager.lib.nut

```diff
@@ -75,14 +75,14 @@ class ConnectionManager {
         server.setsendtimeoutpolicy(errorPolicy, waitPolicy, ackTimeout);
 
         switch (startBehavior) {
-            case START_NO_ACTION:
+            case CM_START_NO_ACTION:
                 // Do nothing
                 break;
-            case START_CONNECTED:
+            case CM_START_CONNECTED:
                 // Start connecting if they ask for it
                 imp.wakeup(0, connect.bindenv(this));
                 break;
-            case START_DISCONNECTED:
+            case SCM_TART_DISCONNECTED:
```

Comment:
typo: constant name should be CM_START_DISCONNECTED

Reply:
Fixed. Thanks!